More user friendly fail message (was NullPointerException)

Darsstar opened the pull request:
Yes, I should never have created "D:\eclipse workspaces" in the first
place.
No, I am not replacing "%20" with spaces, or creating an URI instance.
"D:\eclipse%20workspaces" is a valid folder and I'm not willing to go
down the rabbit hole and see what that turns into.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/framework/9665)
<!-- Reviewable:end -->

hesara commented:
otherwise looks ok to me, whitespace to correct - please update the PR or check the box that allows project maintainers to make edits to the PR

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[server/src/test/java/com/vaadin/tests/CompileTransitionPropertyTest.java, line 37 at r1](https://reviewable.io:443/reviews/vaadin/framework/9665#-KpKH4XF44qPdscIuywt:-KpKH4XF44qPdscIuywu:bkoi2tw) ([raw file](https://github.com/vaadin/framework/blob/729c5c2113627e6bcbf517928c898643f8378827/server/src/test/java/com/vaadin/tests/CompileTransitionPropertyTest.java#L37)):*
> ```Java
>         String file = getClass().getResource("styles.scss").getFile();
>         if (file.contains("%20")) {
>         	fail("path contains spaces, please move the project");
> ```

use spaces instead of tabs

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/9665#-:-KpKH8ACMKKcS5hmX3Vx:bk1meix)*
<!-- Sent from Reviewable.io -->

Darsstar committed:
More user friendly fail message (was NullPointerException)

Yes, I should never have created "D:\eclipse workspaces" in the first
place.
No, I am not replacing "%20" with spaces, or creating an URI instance.
"D:\eclipse%20workspaces" is a valid folder and I'm not willing to go
down the rabbit hole and see what that turns into.

Darsstar commented:
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[server/src/test/java/com/vaadin/tests/CompileTransitionPropertyTest.java, line 37 at r1](https://reviewable.io:443/reviews/vaadin/framework/9665#-KpKH4XF44qPdscIuywt:-KpLAY34xU-PzVrGOwTg:b-896fix) ([raw file](https://github.com/vaadin/framework/blob/729c5c2113627e6bcbf517928c898643f8378827/server/src/test/java/com/vaadin/tests/CompileTransitionPropertyTest.java#L37)):*
<details><summary><i>Previously, hesara (Henri Sara) wrote…</i></summary><blockquote>

use spaces instead of tabs
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/9665)*
<!-- Sent from Reviewable.io -->

hesara commented:
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/9665)*
<!-- Sent from Reviewable.io -->
